src/check_data.py

```diff
@@ -0,0 +1,94 @@
+import toml
+import pandas as pd
+from pathlib import Path
+
+
+def read_toml(path: Path) -> dict:
+    return toml.load(path)
+
+def read_csv(path: Path) -> pd.DataFrame:
+    return pd.read_csv(Path)
+
+def flatten_list_of_dicts(key: str, val: str, data: list) -> dict:
```

Comment:
Perhaps we should just adjust the input file, instead of having to convert it?

Reply:
#2